tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -81,10 +81,7 @@ def models(self, diff_entry):
             return
         model_name, *remaining_path = remaining_path
         if not remaining_path:
-            # A new model or a model deletion is not very interesting by itself
-            # since it usually means that there is a new operation
-            #
-            # We might miss some discrimanator new sub-classes however
+            self.features.append(_MODEL_ADD.format(model_name))
```

Comment:
don't know much about the code, but judging by the original comment, it seems that a model deletion would also make its way into this chunk of code. So if we do add this functionality, I think we should add a check if it's a model that's being added or deleted, instead of just assuming it's a model addition